src/shared/services/activity/activity.service.ts

```diff
@@ -66,22 +75,22 @@ export class ActivityService {
     const options = this.getOptions();
     const url = `${this.activityUrl}/cases/${caseId.join(',')}/activity`;
     return this.http
-      .get(url, options, false)
+      .get(url, options, false, ActivityService.handleHttpError)
       .map(response => response);
   }
 
-  public postActivity(caseId: string, activityType: String): Observable<Activity[]> {
+  public postActivity(caseId: string, activity: String): Observable<Activity[]> {
```

Comment:
`string` not `String` . I know this not you but we could tidy it up.

Reply:
Oh! I thought I'd sorted that one already! 😱

Might have been on a different branch but I'll make sure it's all good here too. Thank you.